blas/unit_test/Test_Blas1_team_mult.hpp

```diff
@@ -368,6 +366,7 @@ int test_team_mult_mv() {
   // view_type_c_ll, Device>(132231,5);
 #endif
 
+  /*
```

Comment:
Loss of coverage? Please add the reasoning for commenting these tests out in a comment.